public/styles.css

```diff
@@ -1,229 +1,226 @@
 body {
-    background: linear-gradient(45deg, #415A85, #415A85);
-    margin: 0;
+  background: linear-gradient(45deg, #415A85, #415A85);
+  margin: 0;
 }
 
 nav {
-    width: 100%;
-    max-width: 440px;
-    height: 70px;
-    margin-left: auto;
-    margin-right: auto;
-    font-family: 'PT Sans';
-    font-size: 20px;
-    font-weight: 700;
-    position: relative;
-    z-index: -100;
```

Comment:
This is what was removed to fix the issue. I was trying to get the shadow to render on top of the background color but behind the links... it didn't work quite right. Now the shadow isn't on top of anything, it's strictly below it, so hey.